assignments/favourite_movie/favourite_movie.js

```diff
@@ -2,6 +2,15 @@ const movies = [];
 
 function favouriteMovie(operation, movie) {
     // Write your code here
+    if(operation==="add"){
+        movies.push(movies);
+    }
+    else if(operation==="remove"){
+        movies.pop(movies);
```

Comment:
Every thing is fine. I think no need to pass the variable because pop delete the variable at end